web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -1,27 +1,59 @@
 <script lang="ts">
+	import { onMount } from 'svelte';
+	import { lazyLoad } from 'unlazy';
 	import { imageLoad } from '$lib/utils/image-load';
 
 	export let url: string;
 	export let altText: string;
 	export let heightStyle: string | undefined = undefined;
 	export let widthStyle: string;
+	export let thumbhash: string | null = null;
 	export let curve = false;
 	export let shadow = false;
 	export let circle = false;
 	let loading = true;
+
+	let imageElement: HTMLImageElement;
+
+	onMount(() => {
+		if (thumbhash != null) {
```

Comment:
```suggestion
		if (thumbhash) {
```

Reply:
I don't think an empty thumbhash is "valid", even if the library doesn't throw an error.